README.md

```diff
@@ -1,2 +1,454 @@
-# spec
-The UCAN spec
+# User Controlled Authorization Network (UCAN) Specification v0.8.0
+
+## Editors
+
+* [Brooklyn Zelenka](https://github.com/expede), [Fission](https://fission.codes)
+
+## Authors
+
+* [Brooklyn Zelenka](https://github.com/expede), [Fission](https://fission.codes)
+* [Philipp Krüger](https://github.com/matheus23), [Fission](https://fission.codes)
+* [Daniel Holmgren](https://github.com/dholms), `<REDACTED>`
+
+# 0. Abstract
+
+User Controlled Authorization Network (UCAN) is a trustless, secure, local-first, user-originated authorization and revocation scheme. It provides public-key verifiable, delegable, expressive, openly extensible [capabilities](https://en.wikipedia.org/wiki/Object-capability_model) by extending the familiar [JWT](https://datatracker.ietf.org/doc/html/rfc7519) structure. UCANs achieve public verifiability via chained certificates, and [decentralized identifiers (DIDs)](https://www.w3.org/TR/did-core/). Verifiable chain compression is enabled via [content addressing](https://en.wikipedia.org/wiki/Content-addressable_storage). UCAN improves on the familiarity and adoptability of schemes like [SPKI/SDSI](https://theworld.com/~cme/html/spki.html) for a web context and native application context, allowing creation and discharge by any agent with a DID, including peer-to-peer beyond traditional cloud computing.
```

Comment:
> Does UCAN improve familiarity and adoptability because agents with with DIDs can create and discharge? 

No, it's more familiar because it's encoded as a JWT. SPKI has its own special Lisp dialect. For example:

```clojure
(public-key
   (rsa-pkcs1-md5
    (e #03#)
    (n
     |ANHCG85jXFGmicr3MGPj53FYYSY1aWAue6PKnpFErHhKMJa4HrK4WSKTO
     YTTlapRznnELD2D7lWd3Q8PD0lyi1NJpNzMkxQVHrrAnIQoczeOZuiz/yY
     VDzJ1DdiImixyb/Jyme3D0UiUXhd6VGAz0x0cgrKefKnmjy410Kro3uW1| )))
```